TOC.md

```diff
@@ -90,6 +91,141 @@
     - [TiDB Troubleshooting Map](/tidb-troubleshooting-map.md)
     - [Troubleshoot Cluster Setup](/troubleshoot-tidb-cluster.md)
     - [Troubleshoot TiDB Lightning](/troubleshoot-tidb-lightning.md)
+=======
```

Comment:
@toutdesuite Please resolve the conflict. Be careful about the version difference.

Reply:
@TomShawn Resolved. PTAL~